### PR TITLE
Set `null` to file path callback 

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
File path callback should be `null` after being used once. This PR aims to add this fix.

## Links
- MINI-3179

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt [hunk 1]
```diff
@@ -36,20 +37,22 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
...
 
-    @Suppress("TooGenericExceptionCaught", "SwallowedException")
+    @Suppress("TooGenericExceptionCaught", "SwallowedException", "LongMethod")
     override fun onShowFileChooser(
         filePathCallback: ValueCallback<Array<Uri>>?,
         fileChooserParams: WebChromeClient.FileChooserParams?,
         context: Context
     ): Boolean {
         try {
+            resetCallback()
```
Comment (corycaywood) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt:47]: The callback is being set to `filePathCallback` on the next line, so seems like maybe you don't need this one here?

**Review by corycaywood — APPROVED** [review]
Looks good 👍  Just left one question, but probably it's okay as it is.

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt (hunk 1 above)
Comment (minh-rakuten) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt:47]: why do we have to reset here?

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt [hunk 2]
```diff
@@ -66,26 +69,33 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
...
         }
+        resetCallback()
     }
 
     /**
      * Can be used when HostApp wants to cancel the file choosing operation.
      */
     fun onCancel() {
-        callback?.onReceiveValue(null)
+        this.callback?.onReceiveValue(null)
```
Comment (Climbatize) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt:93]: adding this is redundant :)

Comment (khairul-alam-licon) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt:92]: Oh, we need it in order to resetting the values to `null` while cancelling operation e.g. when users want to press back button to return to the root state.

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt (hunk 1 above)
Comment (khairul-alam-licon) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt]: yeah, its optional, so I have removed it.

Comment (khairul-alam-licon) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt]: Yeah, I have removed it.

**Review by Climbatize — APPROVED** [review]
LGTM

**Review by minh-rakuten — APPROVED** [review]
👍

**Pull request merged**